posthog/ai/utils.py

```diff
@@ -285,7 +285,9 @@ def call_llm_and_track_usage(
 
         event_properties = {
             "$ai_provider": provider,
-            "$ai_model": kwargs.get("model"),
+            "$ai_model": kwargs.get("model")
+            or getattr(response, "model", None)
+            or "unknown",
```

Comment:
Not sure how I feel about putting "unknown" if we don't know the model. This would trigger a cost model match that won't be correct. I'd rather not put anything in the field.

Reply:
seems like its in streaming its unknown only so think i should actually just match what was there before - will rejig a little to be more minimal